daemon/container_operations.go

```diff
@@ -63,21 +63,13 @@ func (daemon *Daemon) buildSandboxOptions(container *container.Container) ([]lib
 
 	if container.HostConfig.NetworkMode.IsHost() {
 		sboxOptions = append(sboxOptions, libnetwork.OptionUseDefaultSandbox())
-		if len(container.HostConfig.ExtraHosts) == 0 {
```

Comment:
This code is not used for LCOW?

Reply:
Oh, just see @johnstep's comment; so ignore this one; only check if the vendored file ended up in the right commit

Reply:
@johnstep can you confirm the LCOW behavior here?